site/en/workflows.md

```diff
@@ -105,7 +104,10 @@ level, including the confidence value
   * `other_font`
   * `not_a_font`
 
-**Note:** `ocrd-typegroups-classifier` only works on *non-binarized* RGB images.
+**Note:** `ocrd-typegroups-classifier` was trained on a very large and diverse
+dataset, with both geometric and color-space random augmentation (contrast,
+brightness, hue, even compression artifacts and 2 different binarization
+methods), so it works best on the raw, *non-binarized* RGB image.
 
 **Note:** `ocrd-typegroups-classifier` comes with a non-OCR-D CLI that allows
 for the generation of "heatmaps" on the page to visualize which regions of the page
```

Comment:
Note: in the table below, you cannot use markdown hyperlinks, and this breaks table layout. (Cannot make suggestions out of context, sorry.)